ee/observe/explorer.go

```
@@ -0,0 +1,800 @@
+		&summary.Builds,
+		&summary.Updates,
+		&summary.Sessions,
+		&summary.Events,
+		&summary.Platforms,
+	); err != nil {
+		return fmt.Errorf("reading observe summary: %w", err)
+	}
+	return nil
+}
+
+func metricIndex(metrics []MetricSeries) map[string]*MetricSeries {
```

golangci-lint (2.12.2)

[error] func metricIndex is unused

(unused)